DFE-Digital/Dfe.Academies.External
# Code cleanup remove duplicate code

## Type of change

## Expected behaviour
Amended razor pages to call a base method in base page abstract class to remove duplicate code

Moved base method into base page model

Tidied up some comments

## Steps to reproduce issue (if relevant)
n/a

## Steps to test this PR
1. Run Unit Tests
2. Run the application, go through wizard to ensure everything working as expected
3.
4.

## Prerequisites
n/a
